### ISSUE TITLE
Inkstitch Issue

### ISSUE THREAD

**Issue by 128mega** [inkstitch/inkstitch#2272]
Ink/Stitch Version: v2.2.0 (windows)

Traceback (most recent call last):
  File "lib\elements\fill_stitch.py", line 556, in to_stitch_groups
  File "lib\elements\fill_stitch.py", line 590, in do_underlay
  File "lib\debug.py", line 133, in decorated
  File "lib\stitches\auto_fill.py", line 62, in auto_fill
  File "lib\stitches\fill.py", line 163, in intersect_region_with_grating
ZeroDivisionError: float division by zero

**Comment by kaalleen** [comment]
Try to remove small elements from yourr document. Also try to avoid thin fill stitch areas. I haven't seen your design, but think if they are possibly replaceable with a satin column or running stitch.

**Comment by moolight-seashell** [comment]
Please make a effort when you want to post a issue, 

make a title that summarise your problem,
d'ont post issues multiple times,

and check if your somone else have not put your issue already 
https://github.com/inkstitch/inkstitch/issues/2210

**Comment by 128mega** [comment]
Look man, I got 2 separate ink stitch errors on two separate projects and I was prompted to copy and paste the issue here if I occurred it. Why would I not report a bug when I get it? The prompt didn't say to elaborate or anything, it just said to paste the error text. Plus I don't even know how to use github, so I didn't even know I could view other people's issues!

With all the respect in the world, I don't want you to sass me.

**Comment by moolight-seashell** [comment]
github and git is a tool made for manage version off source code, mainly designed from developer for developers so when you d'ont know a tool be super careful to not make anoying things.

The issue section is manly  the only part that is not only for developers,
As you see , the issues section works like a messaging system where you can reply to each other within the same issue.
when you use a chat app you avoid to make a new chat/topic each time you want to send a message, and you avoid to send the same thing multiple time.

You must use the search bar int the main issue page an search your issue to be sure not make duplicate before creating a new one 

when it ask to past your error it mean in the same issue.
when you post multiple issues, you pollute the issue space and make it less clear and readable.


The issues section is not only a bug report section but also a communication section from a programming point of view in general.

The issues can be closed in this case by following a git commit that would solve the bug

In short, when you don't know a tool, you have to be very careful not to make mistakes, especially when you use a public tool

**Comment by moolight-seashell** [comment]
i am just trying to explain how to have a better use of github issues ? im not trying to help just to make pepole understand what they are doing,

**Comment by kaalleen** [comment]
This is due to our lack of taking care for the issues at the moment. But we do invite our users to post their issues here on GitHub. For most of them it is difficult to recognize if their issue is already posted and we do embrace if they are going the extra step to inform us about any traceback message they are receiving.

When I find the time I'll put some work and clean the issue section.

**Comment by kaalleen** [comment]
Duplicate of #1829
